build.gradle

```diff
@@ -1,13 +1,14 @@
 // BEGIN JAVASCRIPT SUPPORT
 plugins {
     id "com.eriwen.gradle.js" version "1.12.1"
-    id "com.github.rundis.buster" version "0.2.4.2"
+    id "com.moowork.node" version "0.9"
 }
 // END JAVASCRIPT SUPPORT
 
 apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: 'findbugs'
+apply plugin: 'com.moowork.node'
```

Comment:
I don't think this is necessary. I'm not certain, though, so if this line is needed please leave it.